fixtures/owncloud/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+version: '3.8'
+
+services:
+  owncloud:
+    image: owncloud/server
+    ports:
+      - "8080:8080"
+    volumes:
+      - owncloud:/mnt/data
```

Comment:
same depedency missing https://github.com/apache/incubator-opendal/pull/3379/files#diff-e4ffbc031a1c0461139947d312a686b57bfa7f17d0082ef30c1171770e7be613L182-L206

Reply:
Thanks for your review, I've updated it.